src/components/Checkout/AutoDepositUpsell.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+	<div
+		v-if="experimentVisible"
+		id="auto-deposit-upsell"
+		class="tw-py-4 tw-px-2 md:tw-p-4 tw-bg-brand-50 tw-rounded tw-mx-auto tw-mb-8"
+		style="max-width: 50rem;"
+	>
+		<div class="tw-flex tw-mb-4">
+			<div class="tw-hidden md:tw-block tw-mr-3">
+				<img :src="imageRequire(`./loan-re-cycle.svg`)" alt="loan to loan relending graphic">
+			</div>
+			<div class="tw-prose">
+				<h3>Want to help even more people?</h3>
+				<p>
+					Do more good by setting aside money each month. It’s easy to set
+					up and you get to choose who you help.
+					<span class="tw-underline tw-cursor-pointer" @click="showLightbox">Learn more</span>.
+				</p>
+				<transition>
```

Comment:
Do we need to specify transition parameters here? Not sure what this defaults to

Reply:
oof will remove.